OrbitService/CaptureStartStopListener.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_SERVICE_CAPTURE_START_STOP_LISTENER_H_
+#define ORBIT_SERVICE_CAPTURE_START_STOP_LISTENER_H_
+
+#include "CaptureEventBuffer.h"
+
+namespace orbit_service {
+
+// This interface is used to propagate the requests
+// to start and stop the capture received by CaptureServiceImpl,
```

Comment:
Super nit: But I think your text should be aligned to 100column line.

Reply:
The rationale here is that when a comment doesn't fit into a smaller number of lines then I put new lines in places that split the sentence the least. Not sure if it's great practice but that's what's going on here.

Reply:
Still feel it looks a bit strange, but nothing I would argue strongly for or against.

Reply:
Rearranged a bit.